### DISCUSSION TITLE
Segmentation fault on updated Fedora system

### DISCUSSION THREAD

**Discussion by andersvi** [cac-t-u-s/om-sharp#351] [Reports] [answered]
moving  [bug-report](https://github.com/cac-t-u-s/om-sharp/issues/350)  from issues

**Comment by andersvi** [comment]
> Type "apropos word" to search for commands related to "word"...
> Reading symbols from om-sharp...

I beleive you can run gdb like this:
  
 `gdb /usr/bin/om-sharp`

Then type `run` when its ready.

Attach the output here, it might show some useful info, probably not

**Comment by andersvi** [comment]
> Latest release of om-sharp, on lastest updated release of Fedora Linux. 

I'm using fc38 still.  Did it work before updating to fc39?

**Comment by smoge** [comment, marked as answer]
If I'm the only one, I think I found the issue. If others had this too, it was a packaging bug.  For me the issue can be closed.

Sorry for the noise, comrades.

https://github.com/cac-t-u-s/om-sharp/issues/350#issuecomment-1946050974

> after removing, I notice that there were files from the last minor update still in the system, Not sure how they remained there but after removing them, no segfault. The project is nice and dandtyin terms of mem leaks ))).
